dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageErrorOnReference.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+import com.google.auto.value.AutoValue;
+import java.net.URL;
+import javax.annotation.Nullable;
+
+/**
+ * Static linkage error caused by a symbol reference. The symbol reference is one of the three
```

Comment:
You don't really need the second sentence here. If we add a fourth type, you don't want to have to remember to edit this doc.